pkg/sqlmigrations/migrations.go

```diff
@@ -810,9 +810,8 @@ func addRootToAdminRole(ctx context.Context, r runner) error {
 	session := r.newRootSession(ctx)
 	defer session.Finish(r.sqlExecutor)
 
-	// We just created the table, so we can use insert.
 	const upsertAdminStmt = `
-					INSERT INTO system.role_members ("role", "member", "isAdmin") VALUES ($1, $2, true)
+					UPSERT INTO system.role_members ("role", "member", "isAdmin") VALUES ($1, $2, true)
```

Comment:
Are you sure you want to upsert? I'd expect `ON CONFLICT DO NOTHING`.

Reply:
I like to make sure the values are exactly as expected. The `isAdmin` column is not part of the index.

Reply:
hum. This means that my insert into the users table (L727) is also iffy. I'm special-handling uniqueness violations to detect when there is a pre-existing `admin` user, but this will fail in the same way. @bdarnell: looks like I'll need to switch to an `UPSERT` for the admin user as well, we can't reliable check that it does not exist. I could check that it exists with exactly the same values though.